one.flipperzero.qFlipper.yml

```diff
@@ -0,0 +1,47 @@
+app-id: one.flipperzero.qFlipper
+runtime: org.kde.Platform
+runtime-version: "5.15-21.08"
```

Comment:
since it seems to support qt6, you could go directly to 6.3

Reply:
I didn't manage yet to build it on qt6, also seems there's an open issue about bad text rendering: https://github.com/flipperdevices/qFlipper/issues/122

Reply:
on the other hand `5.15-22.08` is available.